From 5dc8638872ef0deeae1371a9e2d551920870f76a Mon Sep 17 00:00:00 2001
From: Dave Jiang <dave.jiang@intel.com>
Date: Wed, 15 Jun 2022 16:26:51 -0700
Subject: [PATCH] MAINTAINERS: idxd driver maintainer update
Git-commit: 5dc8638872ef0deeae1371a9e2d551920870f76a
Patch-mainline: v6.0-rc1
References: bsn#161
Modified-by-SEL: No


Add Fenghua as maintainer of the idxd driver.

Signed-off-by: Fenghua Yu <fenghua.yu@intel.com>
Signed-off-by: Dave Jiang <dave.jiang@intel.com>
Link: https://lore.kernel.org/r/20220615232651.177098-1-fenghua.yu@intel.com
Signed-off-by: Vinod Koul <vkoul@kernel.org>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 MAINTAINERS | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index c9e1353788f2..d499b5783696 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -9935,7 +9935,8 @@ S:	Supported
 Q:	https://patchwork.kernel.org/project/linux-dmaengine/list/
 F:	drivers/dma/ioat*
 
-INTEL IADX DRIVER
+INTEL IDXD DRIVER
+M:	Fenghua Yu <fenghua.yu@intel.com>
 M:	Dave Jiang <dave.jiang@intel.com>
 L:	dmaengine@vger.kernel.org
 S:	Supported
-- 
2.34.1

